website/docs/api/classes/apollo_link_src.MeshApolloLink.md

```diff
@@ -147,8 +147,8 @@ ___
 | Name | Type |
 | :------ | :------ |
 | `test` | (`op`: `Operation`) => `boolean` |
-| `left` | `ApolloLink` \| `RequestHandler` |
-| `right?` | `ApolloLink` \| `RequestHandler` |
+| `left` | `ApolloLink` | `RequestHandler` |
```

Comment:
Those `\|` were meant to be escaped right? 
There are here for a union type, not a markdown cell 👀

Reply:
Could you confirm that now it's fine?

Reply:
Looks good now, thanks!